src/frontend/deployment_details.go

```diff
@@ -50,18 +60,14 @@ func loadDeploymentDetails(httpRequest *http.Request) map[string]string {
 		"hostname": podHostname,
 	}).Debug("Loaded deployment details")
 
-	detailsLoadingStatus = DetailsAreLoaded
+	areDeploymentDetailsLoaded = true
 
 	return deploymentDetailsMap
```

Comment:
No need to return anything. The called (`go loadDeploymentDetails()`) is not expecting anything returned. We just have to modify the global map and silently return (which is implicit at the end of the function, so we don't need a `return` at all)

Reply:
Good catch!
I will work on not returning anything. :)